pkg/agent/baker.go

```diff
@@ -787,5 +787,8 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			}
 			return ""
 		},
+		"FIPSEnabled": func() bool {
+			return config.FIPSEnabled
```

Comment:
Linux and windows share the same configuration? and disabling or enabling FIPs will have an impact on both Linux and Windows.

Reply:
As we chat on Teams just now, 'FIPSEnabled' is an AgentPoolProfile property, not an AKS cluster level property at AKS-RP side, so FIPSEnabled (disabling or enabling) has impact only to the nodes of this AgentPool.